starboard/shared/starboard/media/mime_util.cc

```diff
@@ -99,9 +99,20 @@ bool IsSupportedAudioCodec(const ParsedMimeInfo& mime_info) {
       break;
 #if SB_API_VERSION >= 14
     case kSbMediaAudioCodecMp3:
+      if (mime_type.subtype() != "mpeg" && mime_type.subtype() != "mp3") {
+        return false;
+      }
+      break;
     case kSbMediaAudioCodecFlac:
+      if (mime_type.subtype() != "ogg") {
```

Comment:
Flac doesn't have to be in an ogg container. It can also be a native  stream (mime subtype "flac").

Reply:
That's true. .flac isn't listed as an officially supported container (https://developers.google.com/cast/docs/media#media_container_formats), but it looks like the demuxer handles it, so I think it's fine to include it.